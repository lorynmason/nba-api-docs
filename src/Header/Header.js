 import React from 'react';

const Header = () => {
  return (
    <header>
      <h1 className="title">FANTASY BASKETBALL API DOCUMENTATION</h1>
    </header>
  )
}

export default Header