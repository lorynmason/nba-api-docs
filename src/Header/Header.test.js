import React from 'react';
import Header from './Header';
import { shallow } from 'enzyme'


describe('Header', () => {
  const wrapper = shallow(<Header />)
  it('should matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})