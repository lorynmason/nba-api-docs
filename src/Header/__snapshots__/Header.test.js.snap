// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        FANTASY BASKETBALL API DOCUMENTATION
      </h1>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "FANTASY BASKETBALL API DOCUMENTATION",
      },
      "ref": null,
      "rendered": "FANTASY BASKETBALL API DOCUMENTATION",
      "type": "h1",
    },
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          FANTASY BASKETBALL API DOCUMENTATION
        </h1>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "FANTASY BASKETBALL API DOCUMENTATION",
        },
        "ref": null,
        "rendered": "FANTASY BASKETBALL API DOCUMENTATION",
        "type": "h1",
      },
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
